### ISSUE TITLE
UI component: Links not underlined in unordered lists

### ISSUE THREAD

**Issue by mebates** [18F/web-design-standards#1241]
## Issue type
What kind of issue are you working on?
* UI component

## Description
According to the standard, links should be underlined. Working with my developer, we found that the underline disappears when you put a link in an unordered list. I would expect that the link would be underlined regardless of whether or not it is in an ordered or unordered list. 

cc: @dalfaro1 @albertico

**Comment by maya** [comment]
@mebates thanks for filing this issue! We've recently updated the Draft Standards to fix this here: https://github.com/18F/web-design-standards/pull/1215 it's just waiting on review from a member of our team to merge it in and be released. 

cc: @rogeruiz

**Comment by mebates** [comment]
Awesome, thanks @maya ! :tada:

**Comment by maya** [comment]
Fixed in: #1215.
